src/test/java/org/fundacionjala/coding/manuel/movies/CustomerTest.java

```diff
@@ -0,0 +1,62 @@
+package org.fundacionjala.coding.manuel.movies;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Juan Manuel on 3/18/2018.
+ */
+public class CustomerTest {
+    private static final double DELTA = 0;
+
+    /**
+     * First test.
+     */
+    @Test
+    public void testTetTotalAmount() {
+        final int daysRented1 = 5;
+        final int daysRented2 = 10;
+        final double expectedResult = 20.5;
+        Customer customer = new Customer("Manuel");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Manuel" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)